documentation/src/main/antora/modules/ROOT/pages/component-processor.adoc

```diff
@@ -123,7 +123,55 @@ public MyNewData map(final MyData data,
     return createNewData(data);
 }
 ----
+== Support show dynamic output with a condition
 
```

Comment:
Specifiy that that's a studio feature only..
Join screenshot(s) to be more clear on feature.